server/message_hooks_test.go

```diff
@@ -2847,22 +2838,13 @@ func TestUserWillLogin(t *testing.T) {
 		{
 			Name: "UpdateUser failed during autopromotion",
 			User: &model.User{
-				Id:            testutils.GetID(),
-				EmailVerified: false,
+				Id: testutils.GetID(),
 			},
 			UserIsRemote:                       true,
 			AutomaticallyPromoteSyntheticUsers: true,
 			SetupAPI: func(api *plugintest.API) {
 				api.On("UpdateUser", mock.MatchedBy(func(user *model.User) bool {
-					if user.EmailVerified == false {
-						return false
-					}
-
-					if *user.RemoteId != "" {
-						return false
-					}
-
-					return true
+					return *user.RemoteId == ""
```

Comment:
This will panic when RemoteId is nil.  I suggest using `user.IsRemote`

Reply:
Thanks @wiggin77 I pushed a fix for this!